.github/workflows/android_deploy_staging.yml

```diff
@@ -0,0 +1,50 @@
+name: Android - Deploy Staging build to Firebase
+
+on:
+  push:
+    branches:
+      - develop
```

Comment:
@ryan-conway Could you please set `[chore/5-set-up-cicd-with-github-actions](https://github.com/nimblehq/flutter-ic-lydia-ryan/tree/chore/5-set-up-cicd-with-github-actions)` for demo to get the deploy result for this PR? Then once we capture screen after app is uploaded, we can remove it.